Userland/Libraries/LibWeb/Layout/GridFormattingContext.cpp

```diff
@@ -1649,11 +1649,37 @@ CSSPixels GridFormattingContext::containing_block_size_for_item(GridItem const&
 
 AvailableSpace GridFormattingContext::get_available_space_for_item(GridItem const& item) const
 {
-    auto const& column_track = m_grid_columns[item.raw_column()];
-    AvailableSize available_width = column_track.has_definite_base_size ? AvailableSize::make_definite(column_track.base_size) : AvailableSize::make_indefinite();
+    CSSPixels column_width = 0;
+    bool has_columns_with_definite_base_size = false;
+    for (size_t span = 0; span < item.raw_column_span(); span++) {
```

Comment:
I see how loop that finds spanned tracks by an item is repeated across GFC code. I will have to move it into separate function one day.